### PR TITLE
LoginPage 작동 및 UI 구현입니다.

### PR DESCRIPTION
**Pull request by chepchep2**
에레메세지 없는 loginPage입니다. 
뒤쪽에 수정 버전있습니다.
거기에 구현 영상 올리겠습니다.

### CODE REVIEW

**Comment by chepchep2** [comment]
* **#33** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/33?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#32** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/32?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#31** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/31?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#30** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/30?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#29** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/29?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#28** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/28?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#27** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/27?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#26** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/26?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#25** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/25?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#24** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/24?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#23** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/23?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#22** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/22?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#21** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/21?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#20** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/20?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#19** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/19?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#18** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/18?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `feature/introPage`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @chepchep2 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: lib/common/auth_text_input_widget.dart [hunk 1]
```diff
@@ -86,14 +83,13 @@ class AuthTextInputWidget extends StatelessWidget {
...
+                  icon: const Icon(
                     Icons.cancel,
                     size: 20,
-                    color:
-                        isTextNotEmpty ? AppColor.textHint : AppColor.textHint,
+                    color: AppColor.textHint,
                   ),
-                  splashColor: AppColor.transparent,
-                  highlightColor: AppColor.transparent,
+                  splashColor: null,
```
Comment (flowkater) [inline lib/common/auth_text_input_widget.dart:91]: 이거는 꼭 null 을 지정해야하나요?

File: lib/page/login_page.dart [hunk 2]
```diff
@@ -20,15 +22,9 @@ class LoginPage extends StatefulWidget {
...
-  bool isPasswordValid = true;
 
-  @override
-  void initState() {
-    super.initState();
-    emailController.addListener(updateStatus);
-    passwordController.addListener(updateStatus);
-  }
+  bool checkEmail = false;
+  bool checkPassword = true;
```
Comment (flowkater) [inline lib/page/login_page.dart:27]: checkPassword 는 왜 true 인가요?

File: lib/page/login_page.dart [hunk 3]
```diff
@@ -37,11 +33,38 @@ class _LoginPageState extends State<LoginPage> {
...
+  void checkEmailStatus() {
+    final email = emailController.text;
+    if (!validateEmail(email)) {
+      setState(() {
+        checkEmail = false;
+      });
+    } else {
+      setState(() {
+        checkEmail = true;
+        checkPassword = false;
```
Comment (flowkater) [inline lib/page/login_page.dart:45]: emailStatus 에서 checkPassword 를 굳이 false 를 하는 이유가 뭘까요?

File: lib/page/login_page.dart [hunk 4]
```diff
@@ -37,11 +33,38 @@ class _LoginPageState extends State<LoginPage> {
...
+
+  void checkPasswordStatus() {
+    final password = passwordController.text;
+    if (!validatePassword(password)) {
+      setState(() {
+        checkPassword = false;
+      });
+    } else {
+      setState(() {
+        checkPassword = false;
```
Comment (flowkater) [inline lib/page/login_page.dart:59]: 둘다 false 인게 맞나요?

File: lib/page/login_page.dart [hunk 5]
```diff
@@ -37,11 +33,38 @@ class _LoginPageState extends State<LoginPage> {
...
+      });
+      FocusScope.of(context).nextFocus();
+    }
+  }
+
+  void navigateToFindPasswordPage() {
+    Navigator.of(context).push(
+        MaterialPageRoute(builder: (context) => const FindPasswordPage()));
```
Comment (flowkater) [inline lib/page/login_page.dart:67]: 항상 콤마로 끝내기

**Review by flowkater — CHANGES_REQUESTED** [review]
onEditingComplete 로 잘 바꿨는데, 로직이 좀 이상한 것 같아서..
한번 확인 부탁드립니다!

**Comment by chepchep2** [comment]
### Merge activity

* **Mar 17, 8:12 PM EDT**: @chepchep2 started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/30).
* **Mar 17, 8:55 PM EDT**: [Graphite](https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/30) rebased this pull request as part of a merge.
* **Mar 17, 8:56 PM EDT**: @chepchep2 merged this pull request with [Graphite](https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/30).

**Pull request merged**